.github/workflows/release-api-package.yml

```diff
@@ -0,0 +1,77 @@
+name: Release api package
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
```

Comment:
We have at least v4 now :)

Reply:
Changed, and also changed actions/setup-node to version v3.